src/libraries/Microsoft.PowerFx.Core/Binding/BinderUtils.cs

```diff
@@ -1374,16 +1371,12 @@ public static bool TryGetConstantValue(CheckTypesContext context, TexlNode node,
                     var dottedNameNode = node.AsDottedName();
                     if (dottedNameNode.Left.Kind == NodeKind.FirstName)
                     {
-                        DType enumType = DType.Invalid;
-                        if (context.NameResolver.EntityScope?.TryGetNamedEnum(dottedNameNode.Left.AsFirstName().Ident.Name, out enumType) ?? false)
+                        if (context.NameResolver.Lookup(dottedNameNode.Left.AsFirstName().Ident.Name, out NameLookupInfo nameInfo) && nameInfo.Kind == BindKind.Enum)
                         {
-                            if (enumType.TryGetEnumValue(dottedNameNode.Right.Name, out var enumValue))
+                            if (nameInfo.Data is EnumSymbol enumSymbol && enumSymbol.TryGetValue(dottedNameNode.Right.Name, out OptionSetValue osv))
                             {
-                                if (enumValue is string strValue)
-                                {
-                                    nodeValue = strValue;
-                                    return true;
-                                }
+                                nodeValue = osv.ToObject().ToString();
+                                return true;
                             }
                         }
```

Comment:
Why the change? just curious!

Reply:
We were not getting access to Enums when a symbol table is used